autorag/data/corpus/llama_index.py

```diff
@@ -26,11 +26,16 @@ def llama_documents_to_parquet(llama_documents: List[Document],
         Default is False.
     :return: Corpus data as pd.DataFrame
     """
-    doc_lst = list(map(lambda doc: {
-        'doc_id': str(uuid.uuid4()),
-        'contents': doc.text,
-        'metadata': add_essential_metadata(doc.metadata)
-    }, llama_documents))
+    doc_ids = [str(uuid.uuid4()) for _ in llama_documents]
+    doc_lst = [
+        {
+            'doc_id': doc_id,
+            'contents': doc.text,
+            'metadata': add_essential_metadata(doc.metadata, prev_id, next_id)
+        }
+        for doc, doc_id, prev_id, next_id in zip(llama_documents, doc_ids, [None] + doc_ids[:-1], doc_ids[1:] + [None])
```

Comment:
I believe that there is a Llama Document metadata that saves previous and next id.
Maybe, langchain too.
Its loader might automatically detect what is the prev or next node at TextSplitter (NodeParser at LlamaIndex), and saves to metadata to "certain key"
I don't know what is "certain key". Find that and try to use that information.

Because user can input multiple documents to one long list of Llama Documents, so it will occur that other file that has no relationship at all can detect on prev-next augmentor.